File: a/Processor.cpp
class Processor
{
	private:
		string name;
		int price;

	public:
		// construct
		Processor(string name, int price)
		{
			this->name = name;
			this->price = price;
		}

		// default construct
		Processor(){}

		void setName(string name) { this->name = name; }
		void setPrice(int price) { this->price = price; }

		string getName() { return this->name; }
		int getPrice() { return this->price; }

		void printProcessor()
		{
			cout << "Nama Processor : " << this->name << endl;
			cout << "Harga Processor : " << this->price << endl;
		}

		// deconstruct
		~Processor(){}
};

File: b/Processor.py
class Processor():
	__name: str
	__price: int

	def __init__(self, name:str, price:int):
		self.__name = name
		self.__price = price

	def setName(self, name:str): self.__name = name
	def setPrice(self, price:int): self.__price = price

	def getName(self): return self.__name
	def getPrice(self): return self.__price

	def printProcessor(self):
		print("Nama Processor : ", self.__name)
		print("Harga Processor : ", self.__price)